scripts/src/main/resources/scripts/command/cobigen

```diff
@@ -32,13 +32,13 @@ function doSetup() {
 # $@: optional CobiGen arguments
 function doRunCLI() {
   local cobigen_cmd="cobigen"
-  doEcho "Running: ${cobigen_cmd} ${@}"
+  doEcho "Running: ${cobigen_cmd} ${*}"
```

Comment:
Looks good. **Within** echoed quoted strings, only the $* - or ${*}  -  can be used. See below.